verilog/parser/verilog.y

```diff
@@ -5452,6 +5452,8 @@ net_declaration
     { $$ = MakeTaggedNode(N::kNetDeclaration, $1, $2,
                           MakePackedDimensionsNode($3),
                           $4, $5, $6); }
+  | net_type delay3 net_variable_or_decl_assigns ';'
+  { $$ = MakeTaggedNode(N::kNetDeclaration, $1, $2, $3, $4); }
```

Comment:
Let's try to make the fields of `kNetDeclarations` consistent with each other as much as possible.  For example, the one right above this has 6 arguments.  You may insert `nullptr`s to make similar fields line up.  

The consistency will make it easier to access common fields with functions in `CST/*`.

This consistency isn't strictly enforced today because we lack a proper abstract syntax tree, but some day we might get one.